src/server/graphql/mutations/stripeFailPayment.ts

```diff
@@ -18,37 +19,34 @@ export default {
       description: 'The stripe invoice ID'
     }
   },
-  resolve: async (source, {invoiceId}, {serverSecret}) => {
+  resolve: async (_source, {invoiceId}, {serverSecret}) => {
     // AUTH
     if (serverSecret !== process.env.AUTH0_CLIENT_SECRET) {
       throw new Error('Don’t be rude.')
     }
 
     const r = getRethink()
     const now = new Date()
+    const manager = new StripeManager()
 
     // VALIDATION
-    const {
-      amount_due: amountDue,
-      customer: customerId,
-      metadata,
-      subscription,
-      paid
-    } = await stripe.invoices.retrieve(invoiceId)
+    const invoice = await manager.retrieveInvoice(invoiceId)
+    const {amount_due: amountDue, customer, metadata, subscription, paid} = invoice
+    const customerId = customer as string
     let orgId = metadata.orgId
     if (!orgId) {
-      ;({
-        metadata: {orgId}
-      } = await stripe.customers.retrieve(customerId))
+      const customer = await manager.retrieveCustomer(customerId)
+      orgId = customer.metadata.orgid
```

Comment:
Paranoia check: is this the correct casing for this variable coming back from stripe?